docs/reference/monitoring/troubleshooting.asciidoc

```diff
@@ -27,3 +31,26 @@ monitoring data by using {metricbeat} the indices have `-mb` in their names. If
 the indices do not exist, review your configuration. For example, see
 <<monitoring-production>>.
 
+[discrete]
+[[monitoring-troubleshooting-uuid]]
+=== Monitoring data for some {stack} nodes or instances is missing from {kib}
+
+*Symptoms*:
+The *Stack Monitoring* page in {kib} does not show information for some nodes or 
+instances in your cluster.
+
+*Resolution*:
+Verify that the missing items have unique UUIDs. Each {es} node, {ls} node,
+{kib} instance, and Beat instance is considered unique based on its persistent 
+UUID, which is found in its `path.data` directory. Alternatively, you can find
+the UUIDs in the product logs at startup.
+
+For {ls} nodes, you can also determine the UUID from the `id` property returned
+by the {logstash-ref}/monitoring-logstash.html[monitoring APIs root resource].
```

Comment:
For Beats instances, you can also determine the UUID from the `uuid` property returned by the info API's root resource: https://www.elastic.co/guide/en/beats/filebeat/master/http-endpoint.html#_info.

Reply:
Thanks, I've added that info!

Reply:
Similarly for Kibana, one can call the Kibana Status API and look at the `uuid` property returned in the API's response. I couldn't find documentation specifically for the API but it's mentioned as a note over here: https://www.elastic.co/guide/en/kibana/current/access.html#status.

Reply:
Done, thanks!